elastic/check.py

```diff
@@ -356,8 +356,11 @@ def get_instance_config(self, instance):
         # Support URLs that have a path in them from the config, for
         # backwards-compatibility.
         parsed = urlparse.urlparse(url)
-        if parsed[2] != "":
+        if parsed[2] != "" and parsed[2] == '/':
```

Comment:
I fully agree with @hush-hush this would break dashboards and monitors, if we're stripping the uri path, it was intended as reflected by the comment in lines 356-357. We need to understand why the reason why this mechanism isn't working. 

Also, regardless, the url tag whould remaing `scheme://netloc/` without the `path` or we could break things and I reckon elastic is one of those integrations we have to be particularly careful not to break.